CHANGELOG.md

```diff
@@ -1,13 +1,19 @@
-## vUNRELEASED
+## v1.1.0
+
+This release is the first incremental release since 1.0. It includes bugfixes and some UX improvements.
 
 #### New features and UX changes
 
 - Add support for non-numerical UID/GID as specified in the appc spec ([#2159](https://github.com/coreos/rkt/pull/2159)).
```

Comment:
link to the appc spec section where it is defined:

https://github.com/appc/spec/blob/master/spec/aci.md#image-manifest-schema